src/_assets/css/_variables.scss

```
@@ -74,7 +74,7 @@ $google-font-family:        "Product Sans", $font-family-base;
 // Sidenav, toc, and footer
 $font-size-small:         14px;
 
-// TODO: use / override BS defaults named $hX-font-size.
+// TODO: use / override BS defaults named $hX-font-size. #2859
```

I'm thinking it might be good to have the issue number right next to the TODO. Something like this:

```suggestion
// TODO #2859: use / override BS defaults named $hX-font-size.
```

(It was hard to find a precedent for this; my initial search turned up auto-creation of issues from TODOs, but although that might be cool, I'm more concerned about the other direction. https://github.com/phetsims/chipper/issues/815 had some discussion of this.)